apis/Google.Cloud.Logging.Console/README.md

```diff
@@ -28,3 +28,12 @@ The server supports four URLs:
 - `/async-exception` throws an exception from an async method (which
   calls other async methods)
 
+## Log trace correlation
```

Comment:
I would add a small paragraph saying that setting this library does not persist or start or otherwise manipulates trace information, etc...

Reply:
Yes, I though this was the only docs when I first reviewed and only realized later that it was just the Sample docs.

Let's get the actual documentation clean and clear and then we can work on this one, I agree we can add a small explanation and the point to the actual documentation.